src/vivarium/component.py

```diff
@@ -551,6 +573,29 @@ def get_initialization_parameters(self) -> Dict[str, Any]:
             if hasattr(self, parameter_name)
         }
 
+    def create_lookup_tables(self, builder: "Builder") -> None:
+        if (
+            self.name not in builder.configuration
+            or "lookup_tables" not in builder.configuration[self.name]
+        ):
+            return
+        lookup_table_config = builder.configuration[self.name].lookup_tables
+
+        for table_name, table_config in lookup_table_config.items():
+            if "skip_build" in table_config and table_config.skip_build:
```

Comment:
If you don't want to create the lookup tables out of the box. This will mainly be used if you have custom code to create specific lookup tables. You will have to manually trigger the creation of these lookup tables in your custom component. Think unmodeled causes in the mortality component.